source/raylib/raylib_types.d

```diff
@@ -97,3 +145,33 @@ enum Colors
     MAGENTA = Color(255, 0, 255, 255), // Magenta
     RAYWHITE = Color(245, 245, 245, 255), // My own White (raylib logo)
 }
+
+unittest
+{
+    import std.conv;
+    
+    float x = cast(float)(GetRandomValue(-500, 1000) / 7.0f);
+    float y = cast(float)(GetRandomValue(-500, 1000) / 7.0f);
+    float width = cast(float)(GetRandomValue(0, 200) / 7.0f);
+    float height = cast(float)(GetRandomValue(0, 200) / 7.0f);
```

Comment:
Please use explicit literal values, not `GetRandomValue` If a unittest fails, we need to be able to reproduce it. Random numbers are only feasible in unittests when you use an explicit seed value, but I don't see a huge need for it here.